src/index.ts

```diff
@@ -166,7 +169,7 @@ async function addPosts(
         spinner.fail(`${(<Error>error).message}`);
     } finally {
         spinner.stop();
-        browser.close();
+        currentBrowser && currentBrowser.close();
```

Comment:
This works in term of logic but IMO it's not very readable, can you do something like:
```js
if(currentBrowser) currentBrowser.close();
```
or even better using optional chaining:
```js
currentBrowser?.close();
```